### ISSUE TITLE
pytorch再接触 | 元哥的日记

### ISSUE THREAD

**Issue by liyuan97** [liyuan97/liyuan97.github.io#50]
https://leeee.top/2020/pytorch2/ 

 skorch是基于pytorch的外部工具，集成了很多基础功能，使得训练过程变得异常简单。一个net.fit()就能搞定。本次是MSBD5002的第二次作业，作业要求用MLP实现二分类和多分类的任务。多分类其实是对于图片分类，所以我也照着MNIST的CNNdemo改了个CNN的模型，CNN效果会好一点。但是由于数据集本身的原因，准确率没能上90%。

**Comment by wangcmino** [comment]
踩踩。。。

**Comment by liyuan97** [comment]
> 踩踩。。。

评论再看，养成习惯😇

**Comment by amychu131** [comment]
about how the learning rate , just set to 0.1? no need to find the best?

**Comment by amychu131** [comment]
also no need to 
'You may also de- crease gradually during the learning process to enhance convergence.'???

**Comment by liyuan97** [comment]
> about how the learning rate , just set to 0.1? no need to find the best?

learning rate actually I always setting smaller than 0.001, it always works well. I think learning rate just is a speed to approach the convergence, it's no need to find the best if you do not care about speed.
